components/Referenda-Tracks.jsx

```diff
@@ -57,7 +57,7 @@ async function GetTracks(network, wsUrl, setReturnValue) {
     tableData.push(
       <tr key={trackData[0]}>
         <td><b>{trackData[0]}</b></td>
-        <td><b>{origin}</b></td>
+        <td><a href={`#${origin.toLowerCase().replace(" ", "-")}`}><b>{origin}</b></a></td>
```

Comment:
@DrW3RK this line should do the trick in linking to the descriptions provided below from the table.

Reply:
Perfect! Thank you.